lib/madness/document.rb

```diff
@@ -67,6 +67,12 @@ def set_base_attributes_for_directory
 
     def markdown
       @markdown ||= File.read file
+      @markdown = shortlinks(@markdown) if config.shortlinks
+      @markdown
+    end
```

Comment:
This is wrong. Originally the function only did something if the `@markdown` variable was unset.
Now it does something (only partially) every time.

It should probably be converted to

```ruby
@markdown ||= begin
  ...
end
```